backend/routes/taskRoutes.js

```diff
@@ -10,4 +15,18 @@ const router = express.Router();
 // @acess Private
 router.post("/", [authenticateToken, createTask]);
 
-module.exports = router;
\ No newline at end of file
+// @desc     Fetch all tasks
+// @route    GET /api/tasks/
+// @access   Private
+router.get("/", [authenticateToken, getTasks]);
+
+// @desc     Fetch single task
+// @route    GET /api/tasks/task/:id
+// @acesss   Private
+router.get("/task/:id", [authenticateToken, getTasksById]);
```

Comment:
This should preferably just be `/tasks/:id` since the `:id` already denotes a task so it would be redundant to have `/tasks/task/id`

Reply:
Fixed: Changes are reflected in recent commits